db-init.sh

```diff
@@ -0,0 +1,6 @@
+#wait for the SQL Server to come up
+echo "sleeping DB Init script"
+sleep 30
```

Comment:
just want to make sure the sleep only happens locally and not with the Azure database. No delay should be needed with Azure database, right?

Reply:
@rodriguesk yes, this script is only run when running locally and will create an empty database.

The API container will run `flask db migrate` which will then populate the database.